drivers/gpu/drm/rp1/rp1-dsi/rp1_dsi_dsi.c

```diff
@@ -1531,5 +1600,6 @@ void rp1dsi_dsi_stop(struct rp1_dsi *dsi)
 
 void rp1dsi_dsi_set_cmdmode(struct rp1_dsi *dsi, int mode)
 {
+	drm_info(dsi->drm, "Change to %s Mode\n", mode ? "Command" : "Video");
```

Comment:
Left over debug log?

Reply:
Yes... I thought would be harmless to leave in, but since there are hardly any other messages to order it with it's probably not useful to see it.